newton/_src/sensors/warp_raytrace/render.py

```
@@ -65,6 +73,22 @@ def tid_to_pixel_coord(tid: wp.int32, num_worlds: wp.int32, num_cameras: wp.int3
     return world_index, camera_index, py, px
 
 
+@wp.func
+def tid_to_coord_view_priority(tid: wp.int32, num_worlds: wp.int32, num_cameras: wp.int32, width: wp.int32):
```

Ruff (0.14.13)

Unused function argument: `num_worlds`

(ARG001)